kubernetes_env/kube_env.py

```diff
@@ -425,14 +411,161 @@ def handle_filter(args):
     return util.EXIT_SUCCESS
 
 
+################### APPLICATION SPECIFIC FUNCTIONS ###########################
+
+def deploy_application(application):
+    if application == "BK":
+        deploy_bookinfo()
+    elif application == "OB":
+        deploy_online_boutique()
+    elif application == "HR":
+        deploy_hotel_reservation()
+
+def deploy_bookinfo():
+    if check_kubernetes_status() != util.EXIT_SUCCESS:
+        log.error("Kubernetes is not set up."
+                  " Did you run the deployment script?")
+        sys.exit(util.EXIT_FAILURE)
+
+    # launch bookinfo
+    samples_dir = f"{ISTIO_DIR}/samples"
+    bookinfo_dir = f"{samples_dir}/bookinfo"
+    apply_cmd = "kubectl apply -f"
+    book_cmd = f"{apply_cmd} {bookinfo_dir}"
+    cmd = f"{apply_cmd} {YAML_DIR}/bookinfo-services.yaml && "
+    cmd += f"{apply_cmd} {YAML_DIR}/bookinfo-apps.yaml && "
+    cmd += f"{book_cmd}/networking/bookinfo-gateway.yaml && "
+    cmd += f"{book_cmd}/networking/destination-rule-reviews.yaml && "
+    cmd += f"{apply_cmd} {YAML_DIR}/storage.yaml && "
+    cmd += f"{apply_cmd} {YAML_DIR}/istio-config.yaml && "
+    cmd += f"{apply_cmd} {YAML_DIR}/productpage-cluster.yaml "
```

Comment:
Is this still named productpage-cluster?

Reply:
It is named productpage-cluster for BookInfo, and frontend-cluster for the other two applications, who have "frontend" as their root node.